CHANGES.md

```diff
@@ -5,6 +5,7 @@
 ### What's Changed
 
 - feat: refactoring; updates; fixes; bump version by @joe733 in [#283](https://github.com/python-validators/validators/pull/283)
+  * BC break: ValidationFailure renamed to ValidationError,
```

Comment:
```diff
+ - *Breaking Changes*:
+   - `ValidationFailure` renamed to `ValidationError` in [joe733@12ae1f5](https://github.com/joe733/pyvalidators/commit/12ae1f5850555d11e1f1a2c03f597fd10610215a)
  - feat: refactoring; updates; fixes; bump version by @joe733 in [#283](https://github.com/python-validators/validators/pull/283)
-   * BC break: ValidationFailure renamed to ValidationError,
  - build(deps): bump pymdown-extensions from 9.11 to 10.0 by @dependabot in [#273](https://github.com/python-validators/validators/pull/273)
```

Reply:
Changed in https://github.com/python-validators/validators/pull/291/commits/8d72f558ac5894f97b283ad183b4154c9dae4541